src/shared/dumb_components/field.js

```diff
@@ -11,8 +11,8 @@ var Field = (props) =>
   ? <Input {...props} />
   : <div></div>
 
-var Display = ({ name, id, value, description }) =>
-  <p>
+var Display = ({ name, id, value, description, display }) =>
```

Comment:
I really don't think this additional class / toggle belongs in these components. It would be worth allowing a class to be added though